resources/ubuntu-build-all.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
I don't see this in the GitHub UI, please make sure all the scripts have the executable bit set in Git (`git update-index --chmod +x resources/*.sh`)

Reply:
They do.